core/src/main/java/org/jruby/truffle/translator/BodyTranslator.java

```diff
@@ -2036,8 +2163,19 @@ public RubyNode visitPostExeNode(PostExeNode node) {
     public RubyNode visitRationalNode(RationalNode node) {
         final SourceSection sourceSection = translate(node.getPosition());
 
-        // TODO: implement Rational
-        return new FixnumLiteralNode.LongFixnumLiteralNode(context, sourceSection, node.getNumerator());
+        // Translate as Rubinius.privately { Rubinius.convert(a, b) }
```

Comment:
Why privately?
And also why to use Rational.convert to create a Rational literal? Is this spec?